include/tvm/expr.h

```diff
@@ -230,6 +230,10 @@ using Domain = Array<Range>;
 
 // print functions for expr
 TVM_DLL std::ostream& operator<<(std::ostream& os, const NodeRef& n);  // NOLINT(*)
+
+// print a Node to the standard output stream
+TVM_DLL void DumpNode(const NodeRef& n);  // NOLINT(*)
```

Comment:
Let us rename this to

```c++
/*!
 * \brief Dump the node to stderr, used for debug purposes.
 * \param node The input node
 */
TVM_DLL void Dump(const NodeRef& node);
```

Reply:
Fixed. Thanks.

Reply:
did you commit the code?, i cannot see the updated code

Reply:
My bad. Done. Thanks.